<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuctionRoom</class>
 <widget class="QWidget" name="AuctionRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="itemdetailPage">
    <widget class="QFrame" name="frame_bid">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>450</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>300</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Newest bidding</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>410</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_image">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>160</y>
       <width>221</width>
       <height>231</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_overview">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>90</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Overview</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>801</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_logo_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="info_item">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>431</width>
       <height>371</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>41</y>
        <width>39</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Manufacturer:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Starting price:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>BIN price:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_nameItem">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Con mèo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_manufacturerItem">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>N/A</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_startPrice">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>100.000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_binPrice">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>400.000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_description">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>170</y>
        <width>251</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding : 2px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string>Trên bàn là một chú mèo xinh xắn, đen trắng như bức tranh họa với bộ lông mềm mại và mắt to tròn nhìn đầy tò mò. Bộ lông màu đen huyền bí xen kẽ với những vạch trắng tinh tế, tạo nên một vẻ đẹp nổi bật và độc đáo.

Chú mèo này không chỉ là một đối tượng dễ thương mà còn trở thành một đối tác tuyệt vời trong việc quảng cáo và bán hàng. Khi được đặt trên các sản phẩm, hình ảnh của chú mèo tỏa ra sự thoải mái và an ninh, tạo nên một cảm giác thân thiện và gần gũi.

Khả năng mô phỏng động của chú mèo không giới hạn. Từ cử động nhỏ như làm đầu vút lên hay vuốt đuôi, đến những tư thế đáng yêu như quay mình hay nhảy lên khỏi một chiếc hộp nhỏ, chú mèo mang lại không khí vui vẻ và dễ chịu. Những hình ảnh này không chỉ tạo ra sự kết nối với đối tượng mà còn kích thích sự quan tâm từ phía khách hàng.</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_timer">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>410</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>05:00</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_backHome">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_bid">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>500</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Bid</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_nameRoom">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>24</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Room 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_nameOwner">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>abc@xyz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_bin">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>500</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>BIN</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
